### PR TITLE
Jwt token article

### LINKED ISSUE #793: Add topic: JWT authentication
- [x] Write JWT authentication topic
- [x] Add link here: https://documentation.platformos.com/release-notes/26-AUGUST-2019

### PR DESCRIPTION
**Pull request by piotrze**
Fixes: https://github.com/mdyd-dev/nearme-documentation/issues/793

### CODE REVIEW

File: app/views/pages/tutorials/users/signing-in-user-with-jwt-token.liquid [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+#### app/views/pages/api/orders.json.liquid
+
+```liquid
+---
+slug: api/orders
+layout_name: ''
+authorization_policies:
+- modules/signup/api_set_current_user
```
Comment (pavelloz) [inline app/views/pages/tutorials/users/signing-in-user-with-jwt-token.liquid]:
```suggestion
- api_set_current_user
```

File: app/views/pages/release-notes/26-AUGUST-2019.liquid [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  contributors: false
+slug: release-notes/26-AUGUST-2019
+searchable: true
+---
+
+#### {{ page.metadata.description }}
+
+<h4 class="release-note release-note__new">NEW</h4>
+
+* **JWT authentication** - `User` has a new field `jwt_token` which can be then passed as an argument to the new `jwt_decode_and_set_session` mutation. [Authenticate REST json requset with JWT token example](/tutorials/users/authenticate-user-with-jwt-token)
```
Comment (pavelloz) [inline app/views/pages/release-notes/26-AUGUST-2019.liquid]:
```suggestion
* **JWT authentication** - `User` has a new field `jwt_token` which can be then passed as an argument to the new `jwt_decode_and_set_session` mutation. [Authenticate REST json request with JWT token example](/tutorials/users/authenticate-user-with-jwt-token)
```

File: app/views/pages/release-notes/2019/26-AUGUST-2019.liquid [hunk 3]
```diff
@@ -13,7 +13,7 @@ searchable: true
 
 <h4 class="release-note release-note__new">NEW</h4>
 
-* **JWT authentication** - `User` has a new field `jwt_token` which can be then passed as an argument to the new `jwt_decode_and_set_session` mutation
+* **JWT authentication** - `User` has a new field `jwt_token` which can be then passed as an argument to the new `jwt_decode_and_set_session` mutation. /tutorials/users/authenticate-user-with-jwt-token
```
Comment (pavelloz) [inline app/views/pages/release-notes/2019/26-AUGUST-2019.liquid]:
```suggestion
* **JWT authentication** - `User` has a new field `jwt_token` which can be then passed as an argument to the new `jwt_decode_and_set_session` mutation.
```

**Comment by piotrze** [comment]
@pavelloz I've added example in `fetch`

**Review by ignacy — APPROVED** [review]
👍 :shipit:

**Review by pavelloz — APPROVED** [review]

**Pull request merged**